abis/bridge.json

```diff
@@ -780,6 +780,51 @@
       }
     ]
   },
+  {
+    "name": "registerBtcTransfer",
+    "type": "function",
+    "constant": true,
+    "inputs": [
+      {
+        "name": "btcTxSerialized",
+        "type": "bytes"
+      },
+      {
+        "name": "height",
+        "type": "uint256"
+      }, 
+      {
+        "name": "pmtSerialized",
+        "type": "bytes"
+      },  
+      {
+        "name": "derivationArgumentsHash",
+        "type": "bytes32"
+      },
+      {
+        "name": "userRefundAddress",
+        "type": "string"
+      }, 
+      {
+        "name": "lbcAddress",
+        "type": "string"
+      }, 
+      {
+        "name": "lpbtcAddress",
+        "type": "string"
+      }, 
+      {
+        "name": "lpbtcAddress",
```

Comment:
This name is not correct

Reply:
Should be `shouldTransferToContract` I think